### ISSUE TITLE
indent isn't being applied inside parentheses

### ISSUE THREAD

**Issue by garborg** [eslint/eslint#4656]
eslint version:
```sh
eslint -v
v1.10.3
```

test.js:
```js
/*eslint indent: [2, 2]*/

function foo(x) {
  var x = [
      1
  ];
  return (
      x
  );
}
```

eslint output
```sh
eslint test.js

/Users/sean/Desktop/test.js
  5:7  error  Expected indentation of 4 space characters but found 6  indent

✖ 1 problem (1 error, 0 warnings)
```

So it caught the over indentation within brackets, but not within parentheses.

I understand that neither is technically a block, but as the rule is applied in the first case like I'd expected, I'd expect it to apply in both cases.

**Comment by gyandeeps** [comment]
duplicate https://github.com/eslint/eslint/issues/1801

**Comment by garborg** [comment]
Thanks @gyandeeps. Feel free to close if that's how duplicates are handled.

**Comment by ilyavolodin** [comment]
Closing as duplicate of #1801
